scss/sin-core/grid/_simple-grid-mixin.scss

```diff
@@ -9,19 +9,21 @@
 /// @param  {number}  $template-columns [12] - Breakpoint will work up to `$max`
 /// @content
 /// @example scss A layout with three columns
-/// .grid-example{
-///   @include simple-grid(3){
-///     background-color: purple;
-///     padding: 1rem;
+///   @use 'grid/simple-grid-mixin' as *;
+///   .grid-example{
+///     @include simple-grid(3){
+///       background-color: purple;
+///       padding: 1rem;
+///     }
```

Comment:
Code inside an `@example` annotation needs to be nested by two spaces, otherwise SassDoc interprets the code itself as just more SassDoc comments.

Reply:
Aha! Another thing I didn't know, and probably was causing some issues. I have updated all the `@example`s accordingly, and they are all rendering gloriously now!